From: John Zupin <johnz@lunarg.com>
Date: Tue, 23 Jun 2020 16:01:35 -0600
Subject: patch: hlsl prepend glslang to cmake install dir

---
 hlsl/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hlsl/CMakeLists.txt b/hlsl/CMakeLists.txt
index 058a67b..7e195c6 100644
--- a/hlsl/CMakeLists.txt
+++ b/hlsl/CMakeLists.txt
@@ -60,5 +60,5 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(HLSL ALIAS glslang::HLSL)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/HLSLTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/HLSLTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/glslang)
 endif()
