From: John Zupin <johnz@lunarg.com>
Date: Tue, 23 Jun 2020 16:01:35 -0600
Subject: patch: hlsl prepend glslang to install dir

---
 hlsl/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hlsl/CMakeLists.txt b/hlsl/CMakeLists.txt
index 62faa19..6cd1bcc 100644
--- a/hlsl/CMakeLists.txt
+++ b/hlsl/CMakeLists.txt
@@ -48,12 +48,12 @@ endif()
 if(ENABLE_GLSLANG_INSTALL)
     if(BUILD_SHARED_LIBS)
         install(TARGETS HLSL EXPORT HLSLTargets
-                ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-                LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+                ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/glslang
+                LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/glslang
                 RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
     else()
         install(TARGETS HLSL EXPORT HLSLTargets
-                ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+                ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/glslang)
     endif()
-	install(EXPORT HLSLTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+	install(EXPORT HLSLTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/glslang/cmake)
 endif(ENABLE_GLSLANG_INSTALL)
