From: John Zupin <johnz@lunarg.com>
Date: Fri, 31 Jul 2020 13:37:50 -0600
Subject: patch: HLSL cmake install location to cmake/glslang

---
 hlsl/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hlsl/CMakeLists.txt b/hlsl/CMakeLists.txt
index 7d5bc15..3f83ce2 100644
--- a/hlsl/CMakeLists.txt
+++ b/hlsl/CMakeLists.txt
@@ -55,5 +55,5 @@ if(ENABLE_GLSLANG_INSTALL)
         install(TARGETS HLSL EXPORT HLSLTargets
                 ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endif()
-	install(EXPORT HLSLTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+	install(EXPORT HLSLTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/glslang)
 endif()
