From: John Zupin <johnz@lunarg.com>
Date: Tue, 23 Jun 2020 16:05:57 -0600
Subject: patch: OSDependent add glslang path to cmake install dir

---
 glslang/OSDependent/Windows/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glslang/OSDependent/Windows/CMakeLists.txt b/glslang/OSDependent/Windows/CMakeLists.txt
index 882133a..e7299c6 100644
--- a/glslang/OSDependent/Windows/CMakeLists.txt
+++ b/glslang/OSDependent/Windows/CMakeLists.txt
@@ -60,5 +60,5 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(OSDependent ALIAS glslang::OSDependent)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/glslang)
 endif()
