From: John Zupin <johnz@lunarg.com>
Date: Wed, 24 Jun 2020 10:13:21 -0600
Subject: patch: update glslang cmake files install path

---
 glslang/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glslang/CMakeLists.txt b/glslang/CMakeLists.txt
index 14249bf..14ebd79 100644
--- a/glslang/CMakeLists.txt
+++ b/glslang/CMakeLists.txt
@@ -212,7 +212,7 @@ if(ENABLE_GLSLANG_INSTALL)
                 ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
     endif()
 
-    install(EXPORT glslangTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(EXPORT glslangTargets DESTINATION ${CMAKE_INSTALL_LIBDIR}/glslang/cmake)
 
     set(ALL_HEADERS
         ${GLSLANG_HEADERS}
